### DISCUSSION TITLE
ffmpeg runonInt to gstream runOnint

### DISCUSSION THREAD

**Discussion by SlowMonk** [bluenviron/mediamtx#2046] [General]
`runOnInit: ffmpeg -f v4l2 -i /dev/video0 -pix_fmt yuv420p -preset ultrafast -b:v 600k -f rtsp rtsp://10.10.10.68:$RTSP_PORT/test
`-> this work fine 

`runOnInit: gst-launch-1.0 -v v4l2src device=/dev/video0 ! videoconvert ! video/x-raw,format=I420 ! x264enc tune=zerolatency bitrate=1024 speed-preset=ultrafast ! h264parse ! rtph264pay config-interval=1 pt=96 ! rtspclientsink location=rtsp://10.10.10.68:8554/test`
this gives 

```
WARNING: erroneous pipeline: no element "rtspclientsink"
2023/07/11 11:51:54 INF [path cam] runOnInit command exited: command returned code 1
```

**Comment by aler9** [comment]
Hello, this is not related to MediaMTX.
`rtspclientsink` is usually shipped in a dedicated GStreamer package, which in Ubuntu is `gstreamer1.0-rtsp`. Make sure to install that packet.

[a comment by an automated account is not shown]
